generators/generator-eclipseditto/pom.xml

```diff
@@ -41,6 +41,12 @@
 			<version>4.12</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.skyscreamer</groupId>

```

Comment:
Is this a new dependency?

Reply:
As far as I know we have it already as transitive dependency of serveral other libraries

Reply:
I couldn't find any use of it in the code and also no CQ - I will create a CQ for it.